cli/tests/pcluster/config/test_section_cluster.py

```diff
@@ -378,19 +378,13 @@ def test_cluster_section_from_file(mocker, config_parser_dict, expected_dict_par
         ("custom_awsbatch_template_url", "NONE", "NONE", None),
         # Settings
         ("scaling_settings", "test1", None, "Section .* not found in the config file"),
-        ("scaling_settings", "test1,test2", None, "is invalid. It can only contains a single .* section label"),
         ("vpc_settings", "test1", None, "Section .* not found in the config file"),
```

Comment:
Why are we removing the tests for single sections? I mean they should be still valid.

Reply:
This logic has been removed from CfnSection to simplify the code and make the Section construction more linear, independently from it being a single or a multiple section (we can talk about this). A default ``validate()`` method has been added to ``SettingsParam``, performing a more generic check based on the ``max_resources`` attribute of the related Section type. This new logic is tested for ``ebs`` and ``queue`` sections in test_validators.